test/pvp-arena.js

```diff
@@ -1807,4 +1805,330 @@ contract("PvpArena", (accounts) => {
       });
     });
   });
+  describe("#rankingBehaviour", () => {
```

Comment:
In most of these tests you are creating global variables. This is a big no-no anywhere in javascript. Define them as `let` in the describe block, or as `const` if you only use them in the scope they are defined

Reply:
`#` is used to specify methods. Since this is not a method but rather a feature, remove the `#`